### ISSUE TITLE
[4.0] [A11y] Feature request: add 'aria-label' to image dialog box

### ISSUE THREAD

**Issue by Scrabble96** [joomla/joomla-cms#33341]
### Is your feature request related to a problem? Please describe.
If you want to add an aria-label to an image for screen readers to read out then it currently has to be added manually in the code editor. The free Windows screen-reader NVDA does not read the 'title' tag but it does read 'aria-label'.

### Describe the solution you'd like
Add an 'aria-label' field to the image dialog box on the 'General' tab:

![image](https://user-images.githubusercontent.com/36987494/116075683-24067500-a68b-11eb-8c07-f3af341735ad.png)

**Comment by brianteeman** [comment]
> The free Windows screen-reader NVDA does not read the 'title' 

The Alternative description is the one you want not image title.

If thats not working for you then you're probably not in browse mode

https://dequeuniversity.com/screenreaders/nvda-keyboard-shortcuts

**Comment by chmst** [comment]
Closing this as it is not an issue. Images do not need aria-labels.
